Sources/HaCWebsiteLib/Events/ExternalWebsiteEvent.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+
+struct ExternalWebsiteEvent : Event, PostCardRepresentable {
+    let title : String
+    let time : DateInterval
+    let tagLine : String
+    let color : String
+    let hypePeriod : DateInterval
+    let imageURL : String?
+    let location : Location?
+    let facebookEventID : Double?
+    var shouldShowAsUpdate: Bool {
+        get {
+            return self.hypePeriod.contains(Date())
+        }
+    }
+    var postCardRepresentation : PostCard {
+        return PostCard(
+          title: self.title,
+          category: .general,
```

Comment:
Currently, we're using `.general` as a `PostCategory` for all events - this is why I'd like to replace the `enum` with a member.